exporter/awskinesisexporter/config.go

```diff
@@ -21,35 +21,16 @@ import (
 // AWSConfig contains AWS specific configuration such as awskinesis stream, region, etc.
```

Comment:
Minor: let's add these configs in README i.e. https://github.com/atlassian-forks/opentelemetry-collector-contrib/tree/master/exporter/awskinesisexporter